src/app/store/index.ts

```diff
@@ -91,6 +96,8 @@ export const store = configureStore({
 
 export const persistor = persistStore(store);
 
+store.dispatch(setInitialPreferenceData());
```

Comment:
No need to call a function to set the initial preference data

Reply:
How can I get existing preference from store om reload?
@kyranjamie

Reply:
We use `redux-persist`. The `settings` slice is already in the `persistConfig` so it happens automatically.